sympy/assumptions/ask.py

```diff
@@ -463,22 +437,17 @@ def ask_full_inference(proposition, assumptions, known_facts_cnf):
 def register_handler(key, handler):
     """
     Register a handler in the ask system. key must be a string and handler a
-    class inheriting from AskHandler::
-
-        >>> from sympy.assumptions import register_handler, ask, Q
-        >>> from sympy.assumptions.handlers import AskHandler
-        >>> class MersenneHandler(AskHandler):
-        ...     # Mersenne numbers are in the form 2**n - 1, n integer
-        ...     @staticmethod
-        ...     def Integer(expr, assumptions):
-        ...         from sympy import log
-        ...         return ask(Q.integer(log(expr + 1, 2)))
-        >>> register_handler('mersenne', MersenneHandler)
-        >>> ask(Q.mersenne(7))
-        True
+    class inheriting from AskHandler.
+
+    .. note:: Deprecated in Sympy 1.8.
```

Comment:
There is `.. deprecated`:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated